frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/vm-settings-tab.tsx

```diff
@@ -130,6 +194,13 @@ export class VMSettingsTabComponent extends React.Component<VMSettingsTabCompone
               )}
             </FormSelect>
           </FormField>
+          {[
+            ProvisionSource.URL.toString(),
+            ProvisionSource.CONTAINER.toString(),
+            ProvisionSource.DISK.toString(),
+          ].includes(provisionSourceValue) && getProvisionSourceDiskHelpMsg()}
```

Comment:
can `getProvisionSourceDiskHelpMsg` be a component rather than a function ?

Reply:
+1, changed to consts